README.md

```diff
@@ -706,9 +706,12 @@ string `foo` are concatenated into a string `42 foo`:
 When concatenating values into a string, leading and trailing
 whitespace is stripped but whitespace between values is kept.
 
-Unquoted strings also support substitutions of course:
+Quoted or unquoted strings can also concatenate with substitutions of course:
 
     tasks-url : ${base-url}/tasks
+    tasks-url : ${base-url}”tasks:colon-must-be-quoted”
```

Comment:
It looks like some "smart" (curly) quotes snuck in.

Reply:
```suggestion
    tasks-url : ${base-url}"tasks:colon-must-be-quoted"
```

Reply:
Thanks, apparently I shouldn't make PRs on an iPhone. Amusingly, the smart quotes appear to crash github's "commit suggestion" button so I have to fix this from my laptop ... 

![image](https://user-images.githubusercontent.com/218689/58245216-49153f00-7d22-11e9-8fab-e99baa408ef3.png)